content/js/users.md

```diff
@@ -152,6 +152,27 @@ skygear.discover('chpapa').then((ben) => {
 });
 ```
 
+<a name="user-profile"></a>
+## User Profile
+
+Whenever a new user signs up, a user profile is automatically created for
+you to track user information other than their username, email or password.
+You can access the user profile the same way as accessing a record.
+
+``` javascript
+const User = skygear.Record.extend("user");
+
+// get the current user's profile
+var query = new skygear.Query(User);
+query.equalTo("_created_by", skygear.currentUser.id);
```

Comment:
I think querying using `_id` is better than using `_created_by`. `_created_by` itself does not specify it is the user id of that user